django/contrib/admin/static/admin/css/widgets.css

```diff
@@ -90,26 +90,32 @@
     display: block;
     text-indent: -3000px;
     overflow: hidden;
+    cursor: default;
+    opacity: 0.3;
```

Comment:
Good point @ryangallen. But can you please disable hover/focus effect for inactive states of this widget? Currently if user move cursor over the inactive link it changes its color which looks not intuitive in terms of UX.